### ISSUE TITLE
After training there is a periodic pattern in the reconstructed melspectrogram

### ISSUE THREAD

**Issue by krantiparida** [facebookresearch/denoiser#138]
Thanks for making the code public. I used the same architecture and parameters as described in the paper. After training the network I see some periodic lines in the melspectrogram of the reconstruction. I am attaching an image below for reference (the reconstructed melspectrogram is the rightmost one). Can you please suggest me what might be the reason for this? This pattern is consistent across all the predictions.
![noise_wR8qIPb8eWw_280](https://user-images.githubusercontent.com/34489236/200113406-3265e903-2b57-4d1b-8517-8e32833db1e5.jpg)

**Comment by adiyoss** [comment]
Hi @krantiparida,
That is strange, we did not observe anything like that in our models. Did you use the same config file as ours? What model version are you using?

**Comment by YunyangZeng** [comment]
We observed similar issue during fine-tuning Demucs. If you are training with only the waveform loss, my experience is this issue can be alleviated by the multi resolution STFT loss. Here is a paper that explains the potential cause of this tonal artifact issue:  
 [https://arxiv.org/abs/2010.14356](url)
